universal-login-relayer/test/integration/ethereum/WalletContractService.test.ts

```diff
@@ -124,5 +124,53 @@ describe('INT: WalletContractService', () => {
       const signature = signStringMessage(msgHash, Wallet.createRandom().privateKey);
       await expect(walletContractService.isValidSignature(utils.hexlify(utils.toUtf8Bytes(message)), proxyContract.address, signature)).to.be.rejectedWith('Invalid owner provided');
     });
+
+    it('decodes execute', async () => {
+      const newKeyPair = createKeyPair();
+      const msg = {
```

Comment:
Extract to contract/testUtils and can use int executeAddKey

Reply:
👍